modules/eslint-plugin/package.json

```diff
@@ -36,7 +36,7 @@
   },
   "sideEffects": false,
   "dependencies": {
-    "@angular-devkit/schematics": "^14.0.0-rc.0",
+    "@angular-devkit/schematics": "^15.0.0",
```

Comment:
We actually don't need this dependency.
When the dependency is removed, we should also disable NX to re-add it during a build.
This can be done by adding disabling `updateBuildableProjectDepsInPackageJson`, for an example see https://github.com/ngrx/platform/pull/3692

```suggestion
```

Reply:
Thanks for your review. I will update the PR

Reply:
@timdeschryver I updated the PR to reflect your suggested changes

Reply:
Thanks @danielsogl